Need to source ROS before building

Currently if I don't source the right ROS 2 distro before colcon build I'll get the following error:

```
$ colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
Starting >>> crazyflie_interfaces
--- stderr: crazyflie_interfaces                         
CMake Error at CMakeLists.txt:13 (find_package):
  By not providing "Findament_cmake.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "ament_cmake", but CMake did not find one.

  Could not find a package configuration file provided by "ament_cmake" with
  any of the following names:

    ament_cmakeConfig.cmake
    ament_cmake-config.cmake

  Add the installation prefix of "ament_cmake" to CMAKE_PREFIX_PATH or set
  "ament_cmake_DIR" to a directory containing one of the above files.  If
  "ament_cmake" provides a separate development package or SDK, be sure it
  has been installed.


---
Failed   <<< crazyflie_interfaces [2.85s, exited with code 1]

Summary: 0 packages finished [3.11s]
  1 package failed: crazyflie_interfaces
  1 package had stderr output: crazyflie_interfaces
  4 packages not processed
```

This is a bit odd as it shouldn't be necessary... either we can find the cause or we need to mention it in the [installation guide](https://imrclab.github.io/crazyswarm2/installation.html) as that is currently missing.

--- a/docs2/installation.rst
+++ b/docs2/installation.rst
@@ -58,10 +58,16 @@ First Installation
     .. code-block:: bash
 
         cd ../
+        source /opt/ros/DISTRO/setup.bash
         colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
 
     .. note::
        symlink-install allows you to edit Python and config files without running `colcon build` every time.
+
+    .. note::
+       If you install it for the first time, you will see a lot of warnings at first. 
+       As long as the build of the package finish, you can ignore this for now. 
+       
 
 5. Set up Crazyradio
 
@@ -106,6 +112,7 @@ You can update your local copy using the following commands:
     git submodule sync
     git submodule update --init --recursive
     cd ../../
+    source /opt/ros/DISTRO/setup.bash
     colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
 
 

